worth2/main/views.py

```diff
@@ -118,3 +126,34 @@ def dispatch(self, request, *args, **kwargs):
 
         return super(ParticipantSessionPageView, self).dispatch(
             request, *args, **kwargs)
+
+    def get(self, request, *args, **kwargs):
+        allow_redo = False
+        needs_submit = self.section.needs_submit()
+        if needs_submit:
+            allow_redo = self.section.allow_redo()
+        self.upv.visit()
+        instructor_link = has_responses(self.section)
+
+        # Was the form submitted with no values selected?
+        is_submission_empty = False
+        for submission in qm.Submission.objects.filter(user=request.user):
```

Comment:
I addressed this bug and updated my commit - see the variable `quizblocks_on_this_page`.